00_Base/src/repository/TokensRepository.ts

```diff
@@ -38,224 +37,6 @@ export class TokensRepository extends SequelizeRepository<OcpiToken> {
     );
   }
```

Comment:
seems that there are a bunch of linter warnings for unused imports, seems that you can get rid of a bunch of the imports then 👍